internal/sdk/cloudian/sdk.go

```diff
@@ -206,6 +206,31 @@ func (client Client) CreateUser(ctx context.Context, user User) error {
 	}
 }
 
+// GetUser gets a user. Returns an error even in the case of a user not found.
+// This error can then be checked against ErrNotFound: errors.Is(err, ErrNotFound)
+func (client Client) GetUser(ctx context.Context, user User) (*User, error) {
+	// FIXME: Introduce UserId struct and enrich User
+	resp, err := client.newRequest(ctx).
+		SetQueryParams(map[string]string{
+			"groupId": user.GroupID,
+			"userId":  user.UserID,
+		}).
+		Get("/user")
+	if err != nil {
+		return nil, err
+	}
+
+	switch resp.StatusCode() {
+	case 200:
+		return &user, nil
+	case 204:
+		// Cloudian-API returns 204 if the user does not exist
+		return nil, ErrNotFound
+	default:
+		return nil, fmt.Errorf("GET unexpected status. Failure: %w", err)
```

Comment:
Same pattern as the rest, including the status code and not the (here `nil`) `err`

```suggestion
		return nil, fmt.Errorf("error: GET user unexpected status code: %d", resp.StatusCode())
```

Reply:
Fixed